cmd/autoupdate/main.go

```diff
@@ -87,7 +87,9 @@ func main() {
 
 		if !noUpdate && len(newVersionsToCommit) > 0 {
 			commitNewVersions(ctx, newVersionsToCommit)
-			commitPackageVersion(ctx, pckg, latestVersion, f)
+			if (*pckg.Version != latestVersion) {
```

Comment:
linting didn't catch the extra parens, do you need them here?

Reply:
Oh, I forgot that Go doesn't require parens. Fixed :)